filebeat/inputsource/udp/server_test.go

```diff
@@ -40,6 +40,15 @@ type info struct {
 }
 
 func TestReceiveEventFromUDP(t *testing.T) {
+	// Excluding udp6 for now, since it fails in our CI
+	for _, network := range []string{networkUDP, networkUDP6} {
```

Comment:
Did you mean to leave networkUDP6 here still? Comment says it should be excluded and tests are failing because of v6.

Reply:
oops, typo, thanks for catching